src/networks.json

```diff
@@ -753,6 +753,20 @@
     "start": 366482,
     "logo": "ipfs://QmZLQVsUqHBDXutu6ywTvcYXDZG2xBstMfHkfJSzUNpZsc"
   },
+  "595": {
+    "key": "595",
+    "name": "Mandala TC7",
+    "chainId": 595,
+    "network": "testnet",
+    "testnet": true,
+    "multicall": "0x8ce86f733024c1ccae2224f05c11fd50713d6f81",
```

Comment:
Looks good, can you verify and publish the contract here plz :)
https://blockscout.mandala.acala.network/address/0x8cE86f733024c1cCAe2224F05C11FD50713D6F81/contracts

Reply:
I complete the verify and publish for the multicall contract.